_posts/2015-10-26-refugeehackathon.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "#refugeehackathon"
+date: "2015-10-26 12:00:00"
+icon: flag
+categories: hackathon
+authors: ["Jan"]
+---
+
+Lots of refugees have landed in Germany. Along with it
```

Comment:
This is text that goes onto the next line as well:

Lots of refugees have landed in Germany. What comes with this is a lot of challenges.